tests/test_wave_05.py

```diff
@@ -46,22 +43,18 @@ def test_get_goal(client, one_goal):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
 def test_get_goal_not_found(client):
-    pass
     # Act
     response = client.get("/goals/1")
     response_body = response.get_json()
 
-    raise Exception("Complete test")
-    # Assert
-    # ---- Complete Test ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # ---- Complete Test ----
+    # # Assert
+    # # ---- Complete Test ----
+    assert response.status_code == 404
+    assert response_body == {"message":f"goal 1 not found"}
```

Comment:
Since we hard code the `1` in the response, we could take off the `f` and use `"goal 1 not found"`.